doc/whats_new.rst

```diff
@@ -103,7 +118,7 @@ Changelog
 
 New features
 ............
-   - All the Scaler classes but :class:`RobustScaler` can be fitted online by
+   - All the Scaler classes but :class:`preprocessing.RobustScaler` can be fitted online by
```

Comment:
Maybe add a newline after the `......`. This seems to be what is done in most cases in whats_new.rst. Not sure whether it has any impact on the rendered html, probably not.